inc/customizer/site-identity.php

```diff
@@ -0,0 +1,79 @@
+<?php
+/**
+ * Additional site identity customizer options.
+ *
+ * @package Primer
+ * @since   1.4.2
+ */
+
+class Primer_Site_Identity_Options {
+
+	/**
+	 * Class constructor.
+	 */
+	public function __construct() {
+
+		if ( class_exists( 'WPaaS\Log\Components\Plugin' ) && WPaaS\Plugin::is_reseller() ) {
```

Comment:
@jonathanbardo Inside of the hotfixes class do you think it would be best to check `is_gd()` or `is_reseller()`? I was going to stick it in [here](https://github.secureserver.net/WPPaaS/wp-paas-system-plugin/blob/develop/gd-system-plugin/includes/class-hotfixes.php#L124-L127), but it looks different from your [initial PR](https://github.secureserver.net/WPPaaS/wp-paas-system-plugin/commit/0181bcb8ccbd17611cead574e40714099868b8d4).